storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -134,14 +136,16 @@
 
 (defn logviewer-link [host fname secure?]
   (if (and secure? (*STORM-CONF* LOGVIEWER-HTTPS-PORT))
-    (url-format "https://%s:%s/log?file=%s"
-      host
-      (*STORM-CONF* LOGVIEWER-HTTPS-PORT)
-      fname)
-    (url-format "http://%s:%s/log?file=%s"
-      host
-      (*STORM-CONF* LOGVIEWER-PORT)
-      fname)))
+    (UIHelpers/urlFormat "https://%s:%s/log?file=%s"
+      (to-array
+        [host
+        (*STORM-CONF* LOGVIEWER-HTTPS-PORT)
+        fname]))
+    (UIHelpers/urlFormat "http://%s:%s/log?file=%s"
+      (to-array
+        [host
+        (*STORM-CONF* LOGVIEWER-PORT)
+        fname]))))
```

Comment:
Never mind I found my bug.  I was running integration-test and not install.  My bad